_data/students.yml

```diff
@@ -1,5 +1,11 @@
 ﻿# New members please add your credentials to a random position in the file.
 # This is to remove conflicts with other people adding their credentials.
+- name: Harshit Bagla 
+  github: harshitbagla
+  image: HarshitBagla.jpg
+  facebook: harshit.bagla.5
+  twitter: harshitbagla
+  bio: A 16 year old Indian, maths, computer science and Physics enthusiast.

```

Comment:
That's no problem mate @animeshsinghweb we could avalod merge conflicts as shown in the above message.